config/dev.exs

```
@@ -28,6 +28,8 @@ config :movie_wager_api, MovieWagerApi.Endpoint,
 # Do not include metadata nor timestamps in development logs
 config :logger, :console, format: "[$level] $message\n"
 
+config :movie_wager_api, :client, url: "http://localhost:4200"
```

Add a comment here on what this is for.

---

Also this should not be in dev.exs, this should be in secret. Or another config file that is NOT in source.

---

Not sure I follow why this shouldn't be in source. It's not a secret that the app runs at localhost:4200 in development. Why do you feel this should be hidden?

---

Any server configuration (ports, database addresses, login creds) shouldn't be in source. Credentials for the obvious security concerns, the rest because deploying a server should be data/config driven.
The only reason you would want 'safety fallbacks' in source is because someone deployed a server they haven't fully configured, which I think is better to just let it fail with a useful message.

---

But the dev environment isn't a server. I agree that we should remove the client url from `prod.exs` but during development, the app will always be running at `localhost:4200`. The only time it wouldn't would be if the dev purposeful ran it on a different port, in which case they should know to update this.